python/ribasim/ribasim/input_base.py

```diff
@@ -426,7 +430,7 @@ def _save(self, directory: DirectoryPath, input_dir: DirectoryPath, **kwargs):
             getattr(self, field)._save(
                 directory,
                 input_dir,
-                sort_keys=self._sort_keys.get("field", ["node_id"]),
```

Comment:
This was broken, since it got the `"field"` instead of `field`, so it always only sorted by `node_id`. I decided to get rid of the default, since the default is usually not right, and makes it easier to forget to add the sort keys.